haystack/nodes/retriever/_embedding_encoder.py

```diff
@@ -125,6 +157,22 @@ def embed_documents(self, docs: List[Document]) -> List[np.ndarray]:
         passages = [[d.meta["name"] if d.meta and "name" in d.meta else "", d.content] for d in docs]  # type: ignore
         return self.embed(passages)
 
+    def train(self, training_data: List[Dict[str, Any]], batch_size: int = 16) -> None:
+        train_examples: List[InputExample] = []
+        for i in training_data:
```

Comment:
I think we could use a list comprehension here.

Reply:
+1

Reply:
Resolved in 022ccad